<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Login</class>
 <widget class="QWidget" name="Login">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>500</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>47</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>500</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>login</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#Login{
	background-image: url(:/qrc/img.png);
}


</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>40</y>
     <width>471</width>
     <height>261</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="uiqrc.qrc">:/qrc/lib1.gif</pixmap>
   </property>
  </widget>
  <widget class="QToolButton" name="toolButton">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>10</y>
     <width>25</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QToolButton#toolButton
{   
   border:one;
   image: url(:/qrc/min.png);
}
QToolButton#toolButton:hover 
{   
	background-color: rgba(255, 255, 255,200);  
    border-radius: 5px;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="iconSize">
    <size>
     <width>16</width>
     <height>16</height>
    </size>
   </property>
  </widget>
  <widget class="QToolButton" name="toolButton_2">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>10</y>
     <width>25</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QToolButton#toolButton_2{
    border:one;
	image: url(:/qrc/up.png);
}

QToolButton#toolButton_2:hover{	
    border:one;
    background-color: rgba(255, 255, 255,200);  
    border-radius: 5px;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QToolButton" name="toolButton_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>460</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QToolButton#toolButton_4{
     border:one;
     border-radius:10px;
     background-color:rgb(0, 170, 127)
}
QToolButton#toolButton_4:hover{
     background-color:rgba(0,170,127,200)
}</string>
   </property>
   <property name="text">
    <string>注册</string>
   </property>
  </widget>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>300</y>
     <width>471</width>
     <height>121</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QStackedWidget{         
      background-color: rgba(255, 255, 255,200);
      border-bottom-left-radius:20px;
      border-bottom-right-radius:20px;   
}</string>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="page">
    <property name="styleSheet">
     <string notr="true">#page{                             
      border-bottom-left-radius:20px;
      border-bottom-right-radius:20px;       
}</string>
    </property>
    <widget class="QToolButton" name="toolButton_3">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>70</y>
       <width>241</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QToolButton#toolButton_3{
    border:one;
    border-radius: 8px;			
	background-color: rgb(99, 170, 117);
     font-size :15px ; 
    font-weight:lighter;       
}
QToolButton#toolButton_3:hover{   	
	background-color: rgba(99, 170, 117,200);
}</string>
     </property>
     <property name="text">
      <string>Login</string>
     </property>
    </widget>
    <widget class="QToolButton" name="toolButton_5">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>10</y>
       <width>24</width>
       <height>22</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QToolButton#toolButton_5{
    border:one;
	image: url(:/qrc/Account.png);
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QToolButton" name="toolButton_6">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>40</y>
       <width>24</width>
       <height>22</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QToolButton#toolButton_6{	
    border:one;
	image: url(:/qrc/password .png);
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>10</y>
       <width>181</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QComboBox{
   border-radius: 10px;
   border : 2px inset green;
   padding-left: 20px;
}
QComboBox:hover,QComboBox:checked{
   border : 2px inset red;
}
QComboBox::drop-down{   
	border-image: url(:/qrc/update.png);
}
QComboBox:drop-down:on{   	
	border-image: url(:/qrc/update1.png);
}
QComboBox QAbstractItemView{        
	   background-color: rgb(255, 255, 255);
       selection-background-color: rgb(85, 170, 0); 
	   outline: 0px;
}
QComboBox QAbstractItemView::item{
}
</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_2">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>40</y>
       <width>181</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit#lineEdit_2{
   border-radius: 10px;
   border : 2px inset green;
   padding-left: 20px;   
}
QLineEdit#lineEdit_2:hover,QLineEdit#lineEdit_2:checked{
   border : 2px inset red;    
}</string>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_2">
    <property name="styleSheet">
     <string notr="true">#page_2{                               
      border-bottom-left-radius:20px;
      border-bottom-right-radius:20px;       
}</string>
    </property>
    <widget class="QLineEdit" name="lineEdit">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>10</y>
       <width>181</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit#lineEdit{
   border-radius: 10px;
   border : 2px inset green;
   padding-left: 20px;   
}
QLineEdit#lineEdit:hover,QLineEdit#lineEdit_2:checked{
   border-radius: 10px;
   border : 2px inset red;
   padding-left: 20px;   
}</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_3">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>40</y>
       <width>181</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit#lineEdit_3{
   border-radius: 10px;
   border : 2px inset green;
   padding-left: 20px;   
}
QLineEdit#lineEdit_3:hover,QLineEdit#lineEdit_2:checked{
   border : 2px inset red;
}</string>
     </property>
    </widget>
    <widget class="QToolButton" name="toolButton_7">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>10</y>
       <width>24</width>
       <height>22</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QToolButton#toolButton_7{
    border:one;
	image: url(:/qrc/Account.png);
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QToolButton" name="toolButton_10">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>40</y>
       <width>24</width>
       <height>22</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QToolButton#toolButton_10{	
    border:one;
	image: url(:/qrc/password .png);
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QToolButton" name="toolButton_11">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>70</y>
       <width>241</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QToolButton#toolButton_11{
    border:one;
    border-radius: 8px;			
	background-color: rgb(99, 170, 117);
     font-size :15px ; 

}
QToolButton#toolButton_11:hover{		
	background-color: rgba(99, 170, 117,200);
}</string>
     </property>
     <property name="text">
      <string>Sign up</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="uiqrc.qrc"/>
 </resources>
 <connections/>
</ui>
